### PR TITLE
adjust the method of lttng trace data parses

### PR DESCRIPTION
**Pull request by liangzhaowang**
Signed-off-by: Zhaowangx.Liang <zhaowangx.liang@intel.com>

### CODE REVIEW

File: analyzer/parser.py [hunk 1]
```diff
@@ -66,10 +67,10 @@ def init_zipkin_data_by_parent_span_id( parent_span_id, root, event, start_time
             zipkin_data[tmp_key] = event['val']
         return
     else:
-        for span_id in root.keys():
-            if not isinstance(span_id, int):
+        for pid in root.keys():
```
Comment (xuechendi) [inline analyzer/parser.py]: for this one, I think we should use span_id, because we may need to put traces from different pid into one tree.

Comment (liangzhaowang) [inline analyzer/parser.py]: i agree, but the name 'span_id' is already exist in this function, so i want to change a variable name.

Comment (xuechendi) [inline analyzer/parser.py]: maybe change to another one? actually in some trace raw file, pid also will be a existing key name.

Comment (liangzhaowang) [inline analyzer/parser.py]: what do you think of "key"

**Pull request merged**